browser/js/orders/order-detail.factory.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+app.factory('OrderDetailFactory', ($http, $state) => {
+
+  let obj = {}
+
+  let baseUrl = '/api/order-details/'
+
+  let getData = (res) => {
+    return res.data
+  }
+
```

Comment:
Above function can be rewritten as:
let getData = res => res.data

Then below functions can be rewritten without the {} and () if there is a single parameter, no return statement is required either.